sushi-config.yaml

```diff
@@ -144,7 +144,15 @@ menu:
 #     better option (see below).
 # The following are simple examples to demonstrate what this might
 # look like:
-#
+
```

Comment:
Change line 76 to:
```yaml
    - input/examples/*
```
restoring the `*` so that the test case examples build

Reply:
This was done in https://github.com/CancerScreeningCDS/crcsm-cds/pull/16/commits/ebfde6a05aa73e59adfffcac02ae18d9d49cc69d.